config/config.go

```diff
@@ -1506,3 +1517,20 @@ func isValidCookieSize(maxCookieSize int) error {
 	}
 	return nil
 }
+
+type TmaxAdjustments struct {
+	// Flag indicating whether the tmax feature is enabled or not
+	Enabled bool `mapstructure:"enabled"`
+	// The minimum duration needed for a bidder to respond back
+	// PBS will not send an HTTP request to the bidder server, if the time needed for PBS processing, adapter's MakeRequests() implementation, building Prebid headers, and applying GZip compression (if needed) is less than bidder_response_min.
+	// By default bidder_response_min will be set to 0. PBS will always send HTTP request to the bidder server if bidder_response_min is 0
+	BidderResponseMin uint `mapstructure:"bidder_response_min"`
+	// Adjustment factor providing a buffer for network delays between PBS and the bidder server
+	// PBS will subtract the bidder_network_latency_buffer from the endpoint's tmax to account for network delays between PBS and the bidder server
+	// If there is no specific buffer to set, the value of bidder_network_latency_buffer should be set to 0
+	BidderNetworkLatencyBuffer uint `mapstructure:"bidder_network_latency_buffer"`
```

Comment:
Please consider adding `ms` suffix to the annotations for these fields so that it's clear to the user that the value is expected to be in milliseconds.

Reply:
[d28cb6b](https://github.com/prebid/prebid-server/pull/2723/commits/d28cb6b2b61b7c2e1900ba22864d2a743c6ba9d2) adds `ms` suffix